### DISCUSSION TITLE
[임윤혁] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by yhyim0627** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#60] [2024년도 과제 제출] [answered]
![image](https://github.com/user-attachments/assets/3f4ea536-3b05-489f-8605-2bc6b573981f)

**Comment by Jinsun-Lee** [comment, marked as answer]
안녕하세요.
자율주행 통합실습 담당 조교입니다.

설문조사는 과제에 포함된 필수 항목입니다. 아래 링크에서 설문을 완료하시기 바랍니다.
https://forms.gle/DDwJCEf9Zbs3fxFi8

감사합니다.
